doc/source/examples/constructing_universe.ipynb

```diff
@@ -37,7 +37,7 @@
     {
```

Comment:
<p>This simtk warning is also weird for an mda.Universe / nglview call :/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
https://github.com/MDAnalysis/mdanalysis/blob/cef7d3fc566ec313a639a92506f81420da7c9358/package/MDAnalysis/__init__.py#L211

Looks like it's a _get_readers thing that loads up all the available converters :/